specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/2017-11-03-preview/hanaonazure.json

```diff
@@ -0,0 +1,422 @@
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/OperationList"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.HanaOnAzure/hanaInstances": {
+            "get": {
+                "tags": [
+                    "HanaOnAzure"
+                ],
+                "operationId": "HanaInstances_List",
+                "summary": "Gets a list of SAP HANA instances in the specified subscription.",
+                "description": "Gets a list of SAP HANA instances in the specified subscription. The operations returns various properties of each SAP HANA on Azure instance.",
+                "x-ms-examples": {
+                    "List all HANA instances in a subscription": {
+                        "$ref": "./examples/HanaInstances_List.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/HanaInstancesListResult"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HanaOnAzure/hanaInstances": {
+            "get": {
+                "tags": [
+                    "HanaOnAzure"
+                ],
+                "operationId": "HanaInstances_ListByResourceGroup",
+                "summary": "Gets a list of SAP HANA instances in the specified subscription and the resource group.",
+                "description": "Gets a list of SAP HANA instances in the specified subscription and the resource group. The operations returns various properties of each SAP HANA on Azure instance.",
+                "x-ms-examples": {
+                    "List all HANA instances in a resource group": {
+                        "$ref": "./examples/HanaInstances_ListByResourceGroup.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    }
+                ],
+                "responses": {
```

Comment:
Add a default response to capture all the error responses. The model must comply with error contract as per RPC. See the example in the swagger for batch RP.

Reply:
For all APIs this must be added.

Reply:
Hey, I am not sure what you mean. What is RPC? and where can I find exmaple in the swagger for batch RP?

Are you saying I need something like
```
"204": {
            "description": "No Content"
          }
```
for default response?

Reply:
here is an example - https://github.com/Azure/azure-rest-api-specs/blob/current/specification/batch/resource-manager/Microsoft.Batch/2017-09-01/BatchManagement.json#L101

Reply:
done